AnimalsTypes/Fox.cs

```diff
@@ -0,0 +1,29 @@
+﻿using GameEngine.Entities;
+using GameEngine.Helpers;
+using Repository;
+
+namespace AnimalsTypes
+{
+    /// <summary>
+    /// Fox entity.
+    /// </summary>
+    public class Fox : Carnivore
```

Comment:
You have added various animals. Where they required? If not that code should be removed.

Reply:
2. Add possibility to catch imported animals and make sure that they will work in your Savanna’s engine (plugins). I tested it on a new animal. Also there was Rabbit.